### ISSUE TITLE
Update Date/Times before adding Timezone to date fields

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#22381]
In 21.09 we added the class `Task210901UpdateDateTimezones` which goes through our postgres database and adds timezone precision to our date columns.  This is a good thing, but unfortunately, in adding the timezone to the date columns, postgres assumes UTC time, which means that `2020-10-30 09:15:00` becomes `2020-10-30 09:15:00+00`.  

The issue is that when a customer has a non-UTC timezone selected, like `US/Eastern` for example, we stored all dates assuming they were `US/Eastern`.  So in the example above, before the timezone is added, dotCMS reads that as 9:15am on Oct. 30th.  Once the Timezone was added to the column, dotCMS reads it as `1:15pm on Oct. 30th`, causing all dates entered in as content to shift based on the Timezone offset.

Bottom line is that we need to update the dates stored in dotCMS based on the system selected timezone BEFORE we add the timezone precision to the date columns.  In the example above, this means that what was stored as `2020-10-30 09:15:00` (assumed US/Eastern) would become `2020-10-30 13:15:00+00` UTC, which dotCMS then displays correctly as 9:15am on Oct. 30th Eastern Time.

**Comment by bryanboza** [comment]
Fixed, this was tested some many times in the development process...

**Comment by erickgonzalez** [comment]
Only needed for 22.03 LTS
